docs/source/ubuntu-opens-install.rst

```diff
@@ -360,8 +360,8 @@ On a compute node, perform the following steps:
     configuration is outside the scope of this install document.
 
     If you *are* configuring a full BGP mesh you'll need to handle the BGP
-    configuration appropriately. You should consult the relevant
-    documentation for your chosen BGP stack.
+    configuration appropriately. To configure BIRD as a route reflector,
+    follow the instructions in :doc:`bird-rr-config`.
 
```

Comment:
Ahhh - interestingly the ubuntu instructions seem clearer on the subject of whether this step works whether you have an RR or not.  Can we be consistent between the Ubuntu and RHEL ones?
The implication here, though, is that the calico-gen-bird stuff will create per-host config that works for the RR - so why do the RR instructions say that you need to go in and fix up the bird.conf files on each compute host?